modules/datadog-slo/catalog/synthetics_slo.yaml

```diff
@@ -0,0 +1,84 @@
+# The official Datadog API documentation with available query parameters & alert types:
+# https://docs.datadoghq.com/api/v1/monitors/#create-a-monitor
+
+# Synthetic Metrics:
+## https://docs.datadoghq.com/synthetics/metrics/
+
+synthetics-slo-generic:
```

Comment:
Can we upstream this to https://github.com/cloudposse/terraform-datadog-platform/tree/master/catalog

Reply:
we should (and this particular SLO does not include any stage filters, so it's generic)

Reply:
https://github.com/cloudposse/terraform-datadog-platform/blob/master/examples/slo/catalog/synthetic.yaml

Reply:
@Benbentwo  why we need the SLOs in both `terraform-aws-components` and `terraform-datadog-platform`?
Why don't use all of them from `terraform-datadog-platform` as remote files?

Reply:
This is the component that refers to that module. The component includes the surrounding yaml merging, whereas the module is more raw resources